File: DsWorkNet/TestWork/TestWork/Dao/DemoDao.cs
using System;

using Dswork.Core.Db;

using TestWork.Model;

namespace TestWork.Dao
{
	public class DemoDao:BaseDao<Demo, long>
	{
		protected override Type GetEntityClass()
		{
			return typeof(Demo);
		}

		public Demo GetShow(long id)
		{
			return this.ExecuteSelect<Demo>("select", id);
		}
	}
}

File: AndroidWeb/src/testwork/dao/demo/DemoDao.java
/**
 * DEMODao
 */
package testwork.dao.demo;

import org.springframework.stereotype.Repository;
import dswork.core.db.BaseDao;
import testwork.model.demo.Demo;

@Repository
@SuppressWarnings("unchecked")
public class DemoDao extends BaseDao<Demo, Long>
{
	@Override
	public Class getEntityClass()
	{
		return Demo.class;
	}
}